### DISCUSSION TITLE
Website about this project.

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#299] [Ideas]
@ThioJoe A website about this project will be great, it can have links of your demo videos, etc. If you like the idea, I'll be down to help you.

**Comment by ThioJoe** [comment]
Eh not really necessary and would just take up time

**Comment by KendallDoesCoding** [comment]
Ok

**Comment by KendallDoesCoding** [comment]
duplicate of #170
